README.md

```diff
@@ -1,100 +1,103 @@
-- No fancy permissions, minimum [permissions](#permissions-used) used to work.
-- Light on browser, size ~45-50 kilobytes
-- Updates every 2 months
-- Open source and documented code
-
-## Permissions Used
-
-- [Storage Local](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/storage/local) API by Mozilla.
-- Access to [erp.iitkgp.ac.in](https://erp.iitkgp.ac.in) to run script for automatic login
-
-## Installation
-
-- From Firefox/Chrome Stores (recommended)
-  <p align="left">
-      <a href="https://addons.mozilla.org/en-US/firefox/addon/erp-auto-login-iitkgp/">
-          <img src="https://blog.mozilla.org/addons/files/2020/04/get-the-addon-fx-apr-2020.svg" alt="ERP Auto Login - IITKGP logo" height="58">
-      </a>
-      <a href="https://chrome.google.com/webstore/detail/erp-auto-login-iitkgp/hianmedhblomknonbbmalncjjffdligl">
-          <img src="https://storage.googleapis.com/web-dev-uploads/image/WlD8wC6g8khYWPJUsQceQkhXSlv1/UV4C4ybeBTsZt43U4xis.png" alt="ERP Auto Login - IITKGP logo">
-      </a>
-  </p>
+<!-- ABOUT THE PROJECT -->
+## About The Project
+<!-- UPDATE -->
+<div align="center">
+  <a href="https://github.com/metakgp/iitkgp-erp-auto-login">
+    <img width="80%" alt="image" src="./screenshots/cover_new.png">
+  </a>
+</div>
+
+ERP Auto Login is a chrome/ firefox extension which can be added to the browser. The extension stores the users confidentials like username, password and security questions and fills them up automatically whenever the user opens the ERP. It also sends the otp to the email, the user only need to fill up the OTP and can login.
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+<div id="supports"></div>
 
-  > For android you can use extensions on the [Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix&hl=en&gl=US) version of Firefox. [follow this article](https://blog.mozilla.org/addons/2020/09/29/expanded-extension-support-in-firefox-for-android-nightly/) to set it up.
+### Supports:
+<!-- UPDATE -->
+1. Shells
+    * `bash`
+    * `zsh`
+2. OS(s)
+    * any `*nix`[`GNU+Linux` and `Unix`]
 
-- From GitHub
+<p align="right">(<a href="#top">back to top</a>)</p>
 
-  - [Click here](https://github.com/siddsarkar/iitkgp-erp-auto-login-extension/releases) to download the latest release `.xpi` for firefox and `.crx` for chrome
-  - Load the extension in `Firefox` as:
-    1. Open Add-on Manager from options (or) by pressing <kbd>Ctrl/Cmd+Shift+A</kbd> on Widows/Mac
-    2. Click the gear icon ⚙️ in the upper-right area of the Add-on Manager Extensions panel.
-    3. Select Install Add-on from file... from the menu, then find and select the dowwloaded (.xpi) file.
-  - Load the extension in `Chrome` as:
-    1. Goto `chrome://extensions/` in your chrome browser
-    2. Enable developer mode by turning the switch on from the top right corner of the chrome browser
-    3. Click on load unpacked and find and select the dowwloaded (.crx) file.
+## Getting Started
```

Comment:
Rename this section to configuring the development environment or something similar and smaller. or just Building from source.